app/scripts/views/mixins/password-mixin.js

```diff
@@ -39,17 +32,10 @@ define(function (require, exports, module) {
       try {
         var passwordField = this.$('.password');
         if (isVisible) {
-          passwordField.attr('type', 'text').attr('autocomplete', 'off')
-            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
```

Comment:
We still need this and the changes below. No need to make any changes to `setPasswordVisibilityFromButton`